docs/usage/shoot/shoot_workload_identity.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Shoot Workload Identity
+description: Configure access to infrastructure accounts via workload identity instead of static credentials
+---
+
+# Shoot Workload Identity
```

Comment:
Can you add a chapter that it is supported to migrate back and forth between `Secret` and `WorkloadIdentity` as authentication method for existing clusters?